### ISSUE TITLE
Icons not showing in remix in production

### ISSUE THREAD

**Issue by ringuralte** [timolins/react-hot-toast#304]
The toast icons like success and error won't show in production when using react-hot-toast with remix. This is how i initiated it in root.tsx

`  return (
    <html lang="en">
      <head>
        <meta charSet="utf-8" />
        <meta name="viewport" content="width=device-width,initial-scale=1" />
        <Meta />
        <Links />
      </head>
      <body>
        <Outlet />
        <Suspense>
          <Toaster
            toastOptions={{
              success: {
                duration: 4000,
              },
              error: {
                duration: 4000,
              },
            }}
          />
        </Suspense>
        <ScrollRestoration />
        <Scripts />
        <LiveReload />
      </body>
    </html>
  )`

**Comment by lime517** [comment]
Strange - I have the same issue in that the icons don't show. I get a spinner icon instead, but it isn't spinning.

**Comment by ringuralte** [comment]
@lime517  yeah it's the same problem for me as well, it works in local development but production have a spinning icon instead which isn't spinning.

**Comment by tb-b** [comment]
Same here. Showing static spinner on Remix.

**Comment by tb-b** [comment]
This has fixed it for me, though it relies on `remix-utils`

```
import { ClientOnly } from "remix-utils/client-only";

<ClientOnly>
    {() => (
        <Toaster //
           ...
        />
    )}
</ClientOnly>
```

**Comment by lime517** [comment]
@tb-b  - is that fix still working for you? I gave it a shot, and while it improved the situation, I often still get icons that aren't loading.

**Comment by lime517** [comment]
@stuart-so  - our team just uses custom icons. I never found a stable solution for the stock icons with Remix. I've also been looking at [https://ui.shadcn.com/docs/components/sonner](https://ui.shadcn.com/docs/components/sonner) as a potential alternative.
